CODE_REVIEW.md

```diff
@@ -0,0 +1,36 @@
+# Are there any problems or code smells in the app?
```

Comment:
You have added E2E test case screenshot files in this MR.
Please remove that instead you can take a screenshot and add it in MR description.